project/V.scala

```diff
@@ -63,6 +63,7 @@ object V {
       .distinct
 
   // Scala 2
+  // whenever version is removed please add it to MtagsResolver under last supported Metals version
```

Comment:
We could add a test that tries to infer if the map contains all Scala versions and warn if it doesn't, but it will not be 100% proof.

Something like check all the versions between 2.12.1 so that they are either in supported list or in the map.

Would that be helpful?

Reply:
Ok, so now we iterate over all supported versions of Scala that exist and check if they are handled, so a maintainer will get a nice message about which version is missing and where to add it.